### ISSUE TITLE
Database support

### ISSUE THREAD

**Issue by LYF123123** [ageitgey/face_recognition#1332]
* face_recognition version: 1.2.3
* Python version: 3.9.5
* Operating System: Window 10 x64 21H1 (19043.1081)

I know there is an example about web_service, however I only find the method (face_recognition.load_image_file) to load the original pictures. Is it possible that add one method to load images from database( just like mongoDB,pgSQL,mySQL)? If add the support about database, I can easily build one web application.

Sorry for my poor English. Thank you very much.

**Comment by viv2712** [comment]
Yes their is a way I will make pull request on this file you get that how we attach our database with the face recognition programe.

**Comment by viv2712** [comment]
And a piece of advise use python 3.6.0 for better compatibility of using face recognition programme.

**Comment by viv2712** [comment]
Hey your issue is been resolved tell me if their is any problem check the file datset.py first and then go detector.py
